src/reducers/cart-reducer.js

```diff
@@ -0,0 +1,32 @@
+const cartInitialState={
+    cart:[],
+}

```

Comment:
you can keep this initial state in the context itself

Reply:
Will do it while refactoring the code.